src/content/community/language-resources/index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Language resources
+description: Non-english resources to learn about Ethereum
+sidebar: true
+lang: en
+---
+
+# Language resources {#language-resources}
```

Comment:
Great intro section! I think we should make sure we call out the fact we do translate ethereum.org & link to https://ethereum.org/en/languages/ in case people somehow land on this page & aren't aware of this.